src/pages/Welcome/style.css

```diff
@@ -1,18 +1,81 @@
+.welcome-container {
+  align-items: center;
+  background-color: #f2b807;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  width: 100%;
+}
+
+.welcome-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
 .welcome-form {
-  width: 250px;
+  max-width: 700px;
+  padding: 0 30px;
   text-align: left;
-  margin-top: 50px;
+}
+
+.shoppin-cart-image {
+  height: auto;
+  max-width: 100%;
+  padding: 26px 110px;
 }
 
 .welcome-text {
-  margin: 40px 0;
+  margin: 32px 0;
+  font-size: 18px;
+  text-align: center;
 }
 
 .welcome-header {
+  align-items: flex-start;
+  font-weight: 600;
+  font-size: 30px;
+  height: 100px;
   position: relative;
+  text-align: center;
   top: unset;
 }
 
 .center {
   text-align: center;
 }
+
+/** component join-list **/
+.share-token {
+  font-size: 18px;
+  font-weight: 600;
+  width: 100%;
+}
+
+@media (min-width: 1024px) {
+  .welcome-container {
+    justify-content: center;
+    height: 100vh;
```

Comment:
I found a small issue with line 58 when the screen height is small that we might be expected.

![image](https://user-images.githubusercontent.com/8689897/142734212-2e97e7a3-cc5a-48d6-9c31-b90ab3a24353.png)

Some white sections appear, I thought there are solutions:
1. Until where you could review the PR teammates (and pages that they are built), I guess your background is a kind of yellow then you can use this color in your `body` tag.
2. Change in line 58 to `min-height: 100vh`.

What do you think about it?